docs/stream-aggregation.md

```diff
@@ -334,35 +336,190 @@ For example, the following config removes the `:1m_sum_samples` suffix added [to
 
 ## Aggregation outputs
 
-The following aggregation outputs can be put in the `outputs` list at [stream aggregation config](#stream-aggregation-config):
+The aggregations are calculated during the `interval` specified in the [config](#stream-aggregation-config)
+and then sent to the storage.
+
+If `by` and `without` lists are specified in the [config](#stream-aggregation-config),
+then the [aggregation by labels](#aggregating-by-labels) is performed additionally to aggregation by `interval`.
+
+Below are aggregation outputs that can be put in the `outputs` list at [stream aggregation config](#stream-aggregation-config).
```

Comment:
```suggestion
Below are aggregation functions that can be put in the `outputs` list at [stream aggregation config](#stream-aggregation-config).
```

Reply:
@hagen1778 i'm not sure that "function" is the right word for the streaming aggregation algorithm, because throughout this article "output" was used for it.